docs/angular/your-first-app/4-loading-photos.md

```diff
@@ -35,8 +35,8 @@ With the photo array data saved, create a function called `loadSaved()` that can
 ```tsx
 public async loadSaved() {
   // Retrieve cached photo array data
-  const photoList = await Preferences.get({ key: this.PHOTO_STORAGE });
-  this.photos = JSON.parse(photoList.value) || [];
+  const { value } = await Preferences.get({ key: this.PHOTO_STORAGE });
+  this.photos = (value ? JSON.parse(value) : []);
```

Comment:
Let's keep the line consistent with the other packages by adding the type to the end:
```tsx
this.photos = (value ? JSON.parse(value) : []) as UserPhoto[];
```